hack/versions/pkg/diff/godiff_test.go

```diff
@@ -59,6 +59,25 @@ func TestCmpGoStructs(t *testing.T) {
 `,
 			b: `package a
 //a different comment
+`,
+			same:      true,
+			shouldErr: false,
+		},
+		{
+			name: "supported types",
```

Comment:
I see you added support for `interface` Type? is this test case for that that? 
```suggestion
			name: "supported interface types",
```